app/components/elements/forms/collection_radio_buttons_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form.collection_radio_buttons field_name, collection, value_method, text_method do |builder| %>
```

Comment:
Since nothing in this branch is using the new components, how do we know what the useful params are? As a reviewer, it's unclear to me why e.g. `value_method` and `text_method` or any of this is here, because nothing's using any of it. I assume this is because it's lifted from H2 and the follow-up PR will make it clear.

Reply:
Actually, this isn't lifted from H2; it is extracted from my H3 code. (I'm trying the keep the size of the next PR smaller.)